txstratum/api.py

```diff
@@ -17,13 +15,24 @@
     from txstratum.manager import TxMiningManager
 
 
+# Default maximum tx weight allowed to be mined in this server.
+MAX_TX_WEIGHT = 35
```

Comment:
We should create a task on hathor-core to update our method that calculates the tx weight, so it can't be above 35.

Reply:
I think this is just to prevent abusing our service, so I don't think we need to hard cap the tx weight in the full node.

Reply:
Yeah, I agree but there are some legit transactions that won't be able to be mined by this service with this cap, that's the problem I see. But I think it's okay we should just know that this case might happen and the user must resolve this tx elsewhere and propagate to the full node later

Reply:
We haven't seen a transaction with such weight so far. But it is a minimum tx weight, so maybe we can even reduce that number in the future.

Reply:
Yeah, I think it's fine.